src/components/Navbar/Burger.style.js

```diff
@@ -0,0 +1,60 @@
+import styled from 'styled-components'
+import { device } from '../../const/device'
+
+export const StyledBurger = styled.div`
+  width: 2rem;
+  height: 2rem;
+  position: fixed;
+  top: 15px;
+  right: 20px;
+  z-index: 20;
+  display: flex;
+  justify-content: space-around;
+  flex-flow: column nowrap;
+
+  @media ${device.tablet} {
+    display: none;
```

Comment:
with React, this is better to unmount hidden elements from VDOM instead of doing this, but ok we do not have enough time